ptx/sec_limit_analytically.ptx

```diff
@@ -665,10 +708,21 @@
               \end{tikzpicture}
             </latex-image>
             <description>
-
+              <p>
+                Sector of a circle from <xref ref="img_squeeze1a"/> with dotted lines
```

Comment:
I think an xref to an image won't work, because images aren't numbered. But it should be possible to adjust the code in the figure to fix this. I'll post a suggestion on Teams, since i think others can make use of it.